File: src/question/month07/day0710/ms_0710.cpp
#include <iostream>
#include <string>
#include <sstream>

using namespace std;

int main() {
	int n = 1;
	int sum = 0;

	cin >> n;
	cin.get();

	if (n <= 10) {
		string score;

		getline(cin, score);

		stringstream split(score);
		string subject;

		while (split >> subject) {
			sum += atoi(subject.c_str());
		}
		
		double avg = sum / (double)n;
		
		printf("avg : %.1lf\n", avg);
		
		if (avg >= 80)
			cout << "pass" << endl;
		else
			cout << "fail" << endl;
	}
	
	return 0;
}

File: src/question/month07/day0710/ms_0710.cs
using System;

namespace algorithm_10
{
    class ms_0710
    {
        static void Main(string[] args)
        {
            int n = Convert.ToInt32(Console.ReadLine());

            if(n <= 10) {
                int sum = 0;
                string info = Console.ReadLine();
                string[] subjects = info.Split(' ');

                foreach (var subject in subjects) {
                    sum += Convert.ToInt32(subject);
                }

                double avg = sum / (double) n;
                Console.WriteLine($"avg : {avg:.0}");
                Console.WriteLine((avg >= 80)? "pass" : "fail");
            }
        }
    }
}
